src/components/victory-axis.jsx

```diff
@@ -437,11 +439,14 @@ class VAxis extends React.Component {
     // TODO: determine line height ("1.2em") based on font size
     const textString = "" + text;
     const textLines = textString.split("\n");
-    return _.map(textLines, (line, index) => {
+    const HTMLtextLines = _.map(textLines, (line, index) => {
       return index === 0 ?
       (<tspan x={x} key={"text-line-" + index}>{line}</tspan>) :
       (<tspan x={x} dy="1.2em" key={"text-line-" + index}>{line}</tspan>);
     });
+    const heightOfLines = this.props.orientation === "top" ?
+     (textLines.length - 1) * this.style.tickLabels.fontSize * 1.25 : 0;
+    return [heightOfLines, HTMLtextLines];
```

Comment:
please return `heightOfLines` from a separate function rather than returning it in an array here